addon/Feeds/PlacesStatsFeed.js

```diff
@@ -15,6 +17,21 @@ module.exports = class PlacesStatsFeed extends Feed {
       PlacesStatsUpdate(historySize, bookmarksSize)
     ));
   }
+  sendStatsPing() {
+    this.options.metadataStore.asyncGetOldestInsert().then(timestamp => (
+      Promise.all([
+        PlacesProvider.links.getHistorySizeSince(timestamp),
+        this.options.metadataStore.asyncCountAllItems()
+      ]).then(([placesCount, metadataCount]) => {
+        let metadataCoverage = 0;
+        if (placesCount !== 0 && metadataCount !== 0) {
+          metadataCoverage = metadataCount / placesCount * 100;
```

Comment:
Just send both counts as integers in two separate pings

countHistoryURLs
countMetadataURLs